readthedocs/projects/tasks.py

```diff
@@ -390,8 +390,8 @@ def run(
                     'stack': True,
                     'tags': {
                         'build': build_pk,
-                        'project': self.project.slug,
-                        'version': self.version.slug,
+                        'project': self.project.slug if self.project else None,
+                        'version': self.version.slug if self.version else None,
```

Comment:
These are initialized as empty dicts in the init method.

Reply:
This should probably be a code comment, not a GH comment, since those disappear after we ship the code :)

Reply:
:100: